wenv

```diff
@@ -283,10 +283,10 @@ run_wenv_def() {
 wenv_new() {
     local usage="\
 USAGE
-  wenv new [-d] [-i <wenv>] [-h] <name> - Create a new wenv called <name>.
+  wenv new [-d <base_dir>] [-i <wenv>] [-h] <name> - Create a new wenv called <name>.
 
 OPTIONS
-  -d        Set the cwd as the new wenv's base directory (i.e. WENV_DIR).
+  -d        Initialize wenv with <base_dir> as the new wenv's base directory rather than the cwd (i.e. WENV_DIR).
```

Comment:
Expand `cwd` to `current working directory` (seems nicer for less experienced users). Also move the `(i.e. WENV_DIR)` between 'directory' and 'rather' (or just remove it altogether, it's pretty obvious what that does, especially once the new wenv opens in the text editor and you see it set there).